post-processor/alicloud-import/post-processor.go

```diff
@@ -72,15 +72,7 @@ type Config struct {
 	// after the import process has completed. `true` means that we should
 	// leave it in the OSS bucket, `false` means to clean it out. Defaults to
 	// `false`.
-	SkipClean bool              `mapstructure:"skip_clean"`
-	Tags      map[string]string `mapstructure:"tags"`
-	// The description of the image, with a length limit of `0` to `256`
-	// characters. Leaving it blank means null, which is the default value. It
-	// cannot begin with `http://` or `https://`.
-	AlicloudImageDescription        string   `mapstructure:"image_description"`
-	AlicloudResourceGroupId         string   `mapstructure:"resource_group_id"`
-	AlicloudImageShareAccounts      []string `mapstructure:"image_share_account"`
-	AlicloudImageDestinationRegions []string `mapstructure:"image_copy_regions"`
```

Comment:
It is not clear to me why these being removed?

Reply:
FYI: I amended the commit message to clarify that they're redundant, and where the original definition comes from.